pint_server/pint_db_migrate/versions/001_in_the_beginning.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (c) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE about this file by physical or electronic mail,
+# you may find current contact information at www.suse.com
+
+from sqlalchemy import (Table, Column, Date, Integer, Numeric,
+                        String, MetaData, Enum)
+from sqlalchemy.dialects import postgresql
+
+
+meta = MetaData()
+
+server_type = postgresql.ENUM('region', 'update', name='server_type',
+                               metadata=meta)
+
+image_state = postgresql.ENUM('deleted', 'deprecated', 'inactive',
+                             'active', name='image_state', metadata=meta)
```

Comment:
Missing one character of indent to align start of `'active'` under start of `'deleted'`

Reply:
good catch!